src/zc/buildout/easy_install.py

```diff
@@ -37,6 +37,8 @@
 import zc.buildout
 import zc.buildout.rmtree
 import warnings
+import csv
+from pip._internal.utils.misc import FakeFile
```

Comment:
No protection against pip internal changes here.

(FakeFile seems like a trivial StringIO-ish class used only to feed the .dist-info/RECORD file into csv.reader(), and I wonder why can't we feed the file directly?)

Reply:
Good point, will go away